graphics/macgui/macmenu.cpp

```diff
@@ -1290,8 +1314,8 @@ bool MacMenu::checkIntersects(Common::Rect &rect) {
 	return false;
 }
 
-bool MacMenu::mouseClick(int x, int y) {
-	if (_bbox.contains(x, y)) {
+bool MacMenu::mouseClick(int x, int y, bool forceClick) {
```

Comment:
What is the purpose of this `forceClick` flag?

Reply:
bypassing the bbox.contains method, think looks like hack but couldn't find good workround